pkg/api/README.md

```diff
@@ -0,0 +1,139 @@
+# github.com/els0r/goProbe
+
+API documentation for goProbe (auto-generated). The default output is JSON. Results can be pretty-printed by passing URL parameter`pretty=1`
```

Comment:
Is the "Content-Type" header set accordingly? I guess some clients might fail if a JSON response isn't marked as such...

Reply:
Really great catch!